src/cascadia/LocalTests_TerminalApp/TabTests.cpp

```diff
@@ -1242,10 +1258,13 @@ namespace TerminalAppLocalTests
         END_TEST_METHOD_PROPERTIES()
 
         auto page = _commonSetup();
-        page->RenameWindowRequested([&page](auto&&, const winrt::TerminalApp::RenameWindowRequestedArgs args) {
+        page->RenameWindowRequested([&page, this](auto&&, const winrt::TerminalApp::RenameWindowRequestedArgs args) {

```

Comment:
Taking a reference to a stack-local variable for a callback doesn't seem to be valid. Or is `RenameWindowRequested` not a callback function?